frontend/view-edit-client/integrations/integrations.component.tsx

```diff
@@ -109,9 +152,14 @@ const css = `
   height: 1.6rem;
 }
 
-.status {
+& .status {
+  margin: 2.4rem 0;
+}
+
+.status > * {
```

Comment:
What is the > * for here?

Reply:
The > means “look for a direct child.” The * means “any type of element”

https://www.w3schools.com/cssref/css_selectors.asp